File: Iniciante/PHP/exercise_1046.php
<?php

$valores = readline();

list($horaInicial, $horaFinal) = sscanf($valores, "%d %d");

if ($horaFinal === $horaInicial) {
    print_r("O JOGO DUROU 24 HORA(S)\n");
} else if ($horaInicial < $horaFinal) {
    $tempoTotal = $horaFinal - $horaInicial;

    print_r("O JOGO DUROU ".$tempoTotal." HORA(S)\n");
} else if ($horaInicial > $horaFinal) {
    $tempoTotal = $horaFinal + 24 - $horaInicial;

    print_r("O JOGO DUROU ".$tempoTotal." HORA(S)\n");
}

//Concluído!

File: Iniciante/Python/exercise_1046.py
horaInicial, horaFinal = [int(x) for x in input().split()]
horaTotal = 24

if horaInicial > horaFinal:
    horaFinal = horaFinal + 24
    horaTotal = horaFinal - horaInicial
elif horaInicial < horaFinal:
    horaTotal = horaFinal - horaInicial

print(''.join(['O JOGO DUROU ', str(horaTotal), ' HORA(S)']))
# print(f"O JOGO DUROU {horaTotal} HORA(S)")